.editorconfig

```diff
@@ -7,3 +7,251 @@ trim_trailing_whitespace = true
+cpp_space_before_template_args = false                                          ; Do not put space before angle brackets in template arguments﻿
+cpp_space_within_template_args = false                                          ; Do not put space within angle brackets in template arguments﻿
+cpp_space_between_closing_angle_brackets_in_template_args = false               ; Do not put space between closing angle brackets in template arguments﻿
+cpp_space_around_alias_eq = true                                                ; Put space around '=' in alias declaration and namespace alias﻿
+cpp_space_around_deref_in_trailing_return_type = false                          ; Do not put space around '->' in trailing return types﻿
+cpp_space_before_colon_in_inheritance_clause = true                             ; Put space before base types list colon﻿
+cpp_space_after_colon_in_inheritance_clause = true                              ; Put space after base types list colon﻿
+#cpp_space_before_type_parameter_constraint_colon                               ; No override for "Before C++/CLI generic constraint colon﻿" (Unspecified)
+#cpp_space_after_type_parameter_constraint_colon                                ; No override for "After C++/CLI generic constraint colon﻿" (Unspecified)
+cpp_space_after_keywords_in_control_flow_statements = true                      ; Put space before parentheses﻿ of control statements﻿
+cpp_space_between_parentheses_of_control_flow_statements = false                ; Do not put space within parentheses﻿﻿ of control statements﻿
+cpp_space_before_semicolon_in_for_statement = false                             ; Do not put space before semicolon﻿ in 'for' statements
+cpp_space_after_semicolon_in_for_statement = true                               ; Put space after semicolon﻿ in 'for' statements
+cpp_space_before_for_colon = true                                               ; Put space before ':' in range-based for loop﻿
+cpp_space_after_for_colon = true                                                ; Put space after ':' in range-based for loop﻿
+cpp_space_before_colon_in_case = true                                           ; Put space before colon in switch case or label statement﻿
+cpp_space_after_colon_in_case = true                                            ; Put space after colon in switch case or label statement﻿
+cpp_space_around_binary_operator = true                                         ; Put space around binary operator﻿
+cpp_space_around_assignment_operator = true                                     ; Put space around assignment operator﻿
+cpp_space_around_member_access_operator = false                                 ; Do not put space around dot, '->', '.*' and '->.'﻿
+cpp_space_within_parentheses = true                                             ; Put space within any parentheses﻿
```

Comment:
I don't think this is often the case, and if it's the case then the space has been placed there usually for alignment of parameter purposes as AFAIK the default would be not having spaces within parentheses.